### PR TITLE
Migrate tutorial checks to GitHub Actions

### PR DESCRIPTION
**Pull request by gino-m**
@jdbcode @donmccurdy PTAL?

GitHub Actions syntax described here:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions

### CODE REVIEW

File: .github/workflows/check-tutorials.js.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow will do a clean install of node and run tests.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```
Comment (donmccurdy) [inline .github/workflows/check-tutorials.js.yml:18]: Nit: The name field here (`Node.js CI`) and the filename (`check-tutorials.js.yml`) are quite different, whereas in the event that other GitHub Actions are added someday we might want them to match. I'd probably omit the `.js` infix as well.

My suggestion would be just "CI / ci.yml" if we expect to add future tests (on things other than tutorials) to this action, or "Check Tutorials / check-tutorials.yml" if not.

Comment (gino-m) [reply .github/workflows/check-tutorials.js.yml:18]: Iiuc, the `paths` setting is defined at the level of the workflow file. Other checks are currently being added for Landcover Toolkit, which will have a different `paths` value. In that case, perhaps it makes sense to give this workflow a more specific name like "Check Tutorials"?

File: .github/workflows/community-tutorial-checks.js.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow will do a clean install of node and run tests.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Community Tutorial checks
```
Comment (jdbcode) [inline .github/workflows/community-tutorial-checks.js.yml:18]: IIUC this is intended to run checks on the JS Code Editor community tutorials?
If so, I wonder if the name should include some indicator of "JS" to make it more clear and delineate from a (future?) IPYNB check.

File: .github/workflows/community-tutorial-checks.js.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+on:
+  push:
+    branches: [master]
+    paths: ["tutorials/**"]
```
Comment (jdbcode) [inline .github/workflows/community-tutorial-checks.js.yml:23]: Similar to above: if this is for JS tutorials, WDYT about restricting the search to `**.js`
to be more clear/explicit about what is what is being targeted for checks.

File: .github/workflows/community-tutorial-checks.js.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  pull_request:
+    branches: [master]
+    paths: ["tutorials/**"]
```
Comment (jdbcode) [inline .github/workflows/community-tutorial-checks.js.yml:26]: Ditto

File: .github/workflows/community-tutorial-checks.js.yml (hunk 2 above)
Comment (gino-m) [reply .github/workflows/community-tutorial-checks.js.yml:18]: These tests actually run against both. I inherited the *.js.* from the examples, but I suppose it's not relevant here. I'll rename instead.

File: .github/workflows/community-tutorial-checks.js.yml (hunk 3 above)
Comment (gino-m) [reply .github/workflows/community-tutorial-checks.js.yml:23]: See above.

File: .github/workflows/community-tutorial-checks.js.yml (hunk 4 above)
Comment (gino-m) [reply .github/workflows/community-tutorial-checks.js.yml:26]: Same here.

File: .github/workflows/community-tutorial-checks.js.yml (hunk 2 above)
Comment (donmccurdy) [reply .github/workflows/community-tutorial-checks.js.yml:18]: Continuing comment from https://github.com/google/earthengine-community/pull/364#discussion_r654590176, it's worth noting that this file could (if desired) contain multiple jobs, i.e. one that checks the JS tutorials and another for IPYNB. So the workflow could be given a generic name ("CI") and have jobs with more specific names ("JS Tutorials", "PY Tutorials").

That said... I'm fine with either approach, please feel free to ignore my earlier comment. :)

File: .github/workflows/check-tutorials.js.yml (hunk 1 above)
Comment (donmccurdy) [reply .github/workflows/check-tutorials.js.yml]: SGTM, whichever you prefer – continuing thread in https://github.com/google/earthengine-community/pull/364/files#r655717853.

File: .github/workflows/community-tutorial-checks.js.yml (hunk 3 above)
Comment (jdbcode) [reply .github/workflows/community-tutorial-checks.js.yml]: Ack

File: .github/workflows/community-tutorial-checks.js.yml (hunk 2 above)
Comment (jdbcode) [reply .github/workflows/community-tutorial-checks.js.yml]: SGTM

**Review by jdbcode — APPROVED** [review]

**Review by donmccurdy — APPROVED** [review]

**Pull request merged**